include/seqan3/alphabet/aminoacid/aa20.hpp

```diff
@@ -81,180 +81,32 @@ namespace seqan3
  *
  * \snippet test/snippet/alphabet/aminoacid/aa20.cpp construction
  */
-
-struct aa20
+class aa20 : public aminoacid_base<aa20, 27>
```

Comment:
why is the second template parameter (size) 27 for aa20?

Reply:
good catch